diff --git a/src/libime/core/kenlm/util/double-conversion/utils.h b/src/libime/core/kenlm/util/double-conversion/utils.h
index 41078b6..3b16e46 100644
--- a/src/libime/core/kenlm/util/double-conversion/utils.h
+++ b/src/libime/core/kenlm/util/double-conversion/utils.h
@@ -132,7 +132,7 @@ int main(int argc, char** argv) {
     defined(__ARMEL__) || defined(__avr32__) || defined(_M_ARM) || defined(_M_ARM64) || \
     defined(__hppa__) || defined(__ia64__) || \
     defined(__mips__) || \
-    defined(__loongarch__) || \
+    defined(__loongarch__) || defined(__sw_64__) || \
     defined(__nios2__) || defined(__ghs) || \
     defined(__powerpc__) || defined(__ppc__) || defined(__ppc64__) || \
     defined(_POWER) || defined(_ARCH_PPC) || defined(_ARCH_PPC64) || \
