From: Shengjing Zhu <zhsj@debian.org>
Date: Thu, 12 Mar 2020 13:18:15 +0800
Subject: use kenlm from debian dir

Last-Update: 2024-01-06
---
 src/libime/core/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/libime/core/CMakeLists.txt b/src/libime/core/CMakeLists.txt
index 89dd191..a7a54be 100644
--- a/src/libime/core/CMakeLists.txt
+++ b/src/libime/core/CMakeLists.txt
@@ -9,10 +9,12 @@ foreach(f ${__KENLM_SRCS})
     endif()
 endforeach()
 
+find_package(double-conversion)
+
 add_library(kenlm STATIC ${KENLM_SRCS})
 target_include_directories(kenlm PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/kenlm>)
 target_compile_definitions(kenlm PUBLIC -DKENLM_MAX_ORDER=3 PRIVATE -DNDEBUG)
-target_link_libraries(kenlm PUBLIC Boost::boost PkgConfig::ZSTD)
+target_link_libraries(kenlm PUBLIC Boost::boost PkgConfig::ZSTD PUBLIC double-conversion::double-conversion)
 set_target_properties(kenlm PROPERTIES
   CXX_STANDARD 11
   POSITION_INDEPENDENT_CODE ON)
